README.md

```diff
@@ -25,10 +25,6 @@
 * `DOCKER_REPO` - The repo in DOCKER_REGISTRY_HOST to store gold master build images (changing this also requires changing jenkins_build_template.xml and existing build templates in jenkins). This has no default.
 * `DYNO_DEFAULT_SIZE` - The default dyno size to use. The set default is `scout` if no other is specified.
 
-### Logging & Metric Information
-* `PROMETHEUS_METRICS_URL` - The url to connect to for metric information stored in prometheus. This has no default.
-* `INFLUXDB_METRICS_URL` - The URL to the influxdb that holds running metrics for service, count and other http information. This has no default.
-
```

Comment:
If these env are not available it will go to the region api for these, its backwards compatible with them however, if they exist it will use them.